### PR TITLE
Web Hosting - Percona Server for MySQL migration informations

### PR DESCRIPTION
**Pull request by fabienOVH**

### CODE REVIEW

**Comment by fabienOVH** [comment]
Wait a GO from Guillaume Fouchaux for publishing the guide

**Comment by OVH-Team-Guides** [comment]
@fabienOVH , translations will be added by @Kazarus4 on this branch before guide publication.

**Comment by Kazarus4** [comment]
All translations are done.

Need proof by : 

- @tcpdumpfbacke for DE and EN subs
- @OVH-Team-Guides for latin and PL subs

File: pages/web_cloud/web_hosting/percona_server_mysql_migration_info/guide.pl-pl.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+### Zakończenie
+
+Migracja MySQL z Oracle na Percona Server dla MySQL ma na celu zwiększenie wydajności i stabilności baz danych współdzielonych w OVHcloud. Aktualizacja ta ma być transparentna dla użytkowników MySQL, bez przerwy w działaniu usługi i bez zmiany interfejsu użytkownika. Gwarantujemy, że migracja odbędzie się bez komplikacji, aby utrzymać ciągłość i jakość usług OVHcloud.
+
+## Sprawdź również
+
+[Tworzenie bazy danych na hostingu](/pages/web_cloud/web_hosting/sql_create_database).
+
+Jeśli chcesz otrzymywać wsparcie w zakresie konfiguracji i użytkowania Twoich rozwiązań OVHcloud, zapoznaj się z naszymi [ofertami pomocy](https://www.ovhcloud.com/pl/support-levels/).
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/web_hosting/percona_server_mysql_migration_info/guide.pl-pl.md]:
```suggestion
Jeśli chcesz otrzymywać wsparcie w zakresie konfiguracji i użytkowania Twoich rozwiązań OVHcloud, zapoznaj się z naszymi [ofertami pomocy](/links/support).
```

**Review by OVH-Team-Guides — APPROVED** [review]
One fix only.
ES, IT, PL, PT are good.

**Review by bs-ovh — APPROVED** [review]

**Comment by Kazarus4** [comment]
Ready to Merge when G.Fouchaux will give his "GO". @fabienOVH could you tell us when G.Fouchaux will give his "GO" plz ?

**Comment by fabienOVH** [comment]
@Kazarus4 yes I warned Guillaume F. about that so he will put his "go" in this conversation when the time comes

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @fabienOVH

**Pull request merged**